MAINTAINERS.yml

```diff
@@ -357,6 +357,18 @@ CMSIS-NN integration:
   labels:
     - "area: CMSIS-NN"
 
+Coding Guidelines:
+  status: maintained
+  maintainers:
+    - keith-zephyr
```

Comment:
Note - I'm not trying to take over coding guidelines, but I didn't want to add more burden to existing maintainers.